<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeTab</class>
 <widget class="QWidget" name="AttributeTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttributeTab</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="typeCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="defaultLabel">
     <property name="text">
      <string>Default:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="defaultEdit"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="isCompositeBox">
        <property name="locale">
         <locale language="English" country="UnitedKingdom"/>
        </property>
        <property name="text">
         <string>Is Composite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isDerivedBox">
        <property name="text">
         <string>Is Derived</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isDerivedUnionBox">
        <property name="text">
         <string>Is Derived Union</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isIDBox">
        <property name="text">
         <string>Is ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isStaticBox">
        <property name="text">
         <string>Is Static</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isReadonlyBox">
        <property name="text">
         <string>Is Readonly</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="aggregationCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="aggregationLabel">
     <property name="text">
      <string>Aggregation:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
